app/styles/_scroll-shadows-horizontal.less

```diff
@@ -0,0 +1,36 @@
+// Horizontal scroll drop shadows
+// Adds drop shadows to left and right edges that display when scrollable content is hidden
+// Test cases http://codepen.io/sg00dwin/pen/GWWRLR/
+@scroll-shadows-horizontal-bg:         rgba(255,255,255,1);
+@bg-color:                             @scroll-shadows-horizontal-bg;
+
+.scroll-shadows-horizontal(@bg-color: @scroll-shadows-horizontal-bg) {
```

Comment:
<img width="1391" alt="screen shot 2017-04-26 at 4 26 09 pm" src="https://cloud.githubusercontent.com/assets/895728/25455479/f78f9fdc-2a9d-11e7-80d4-92fcc226b96d.PNG">

* The aforementioned blue background.
* The outer border is missing at desktop resolutions.
* The shadows aren't being fully masked (there is a ever-so-slight ghosting of the shadows).